Move.java

```diff
@@ -0,0 +1,53 @@
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * class Move that has a boolean method moveFile
+ *
+ * @author Milena Vallejos
+ * @date May 19
+ */
+public class Move {
+    /**
+     * Method to Move file from a source origin fromFile
+     * to a destination toFile
+     *
+     * @param fromFile is a String (source file path)

```

Comment:
Same here review the observations made in the methods documentation of the previous code review